velox/connectors/hive/storage_adapters/hdfs/CMakeLists.txt

```diff
@@ -22,8 +22,9 @@ if(VELOX_ENABLE_HDFS)
     PRIVATE
     HdfsFileSystem.cpp
     HdfsReadFile.cpp
-    HdfsWriteFile.cpp)
-  velox_link_libraries(velox_hdfs Folly::folly ${LIBHDFS3} xsimd)
+    HdfsWriteFile.cpp
+    hdfs_internal.cc)
+  velox_link_libraries(velox_hdfs Folly::folly xsimd arrow)
```

Comment:
Are linking against arrow to also link against libhdfs? Or is this just for the arrow headers? 

Also is it assured that arrow is enabled when `VELOX_ENABLE_HDFS` is used? Otherwise that needs t obe added to the root CMakeLists.txt.

Reply:
@assignUser Yes. We need the arrow to pass the compile. I have enabled arrow when enable `VELOX_ENABLE_HDFS`?